cmd/fluent-bit/loki.go

```diff
@@ -115,12 +118,13 @@ func autoLabels(records map[string]interface{}, kuberneteslbs model.LabelSet) er
 		return errors.New("kubernetes labels not found, no labels will be added")
 	}
 
-	replacer := strings.NewReplacer("/", "_", ".", "_", "-", "_")
+	keyReplacer := strings.NewReplacer("/", "_", ".", "_", "-", "_")
```

Comment:
that could be global too while we are at it.

Reply:
Yeah initially made that global but got error https://cloud.drone.io/grafana/loki/3437/1/2

Reply:
not sure what you mean moving this to a new global variable, should be fine ?

Reply:
For example if the log message has `12.2 foo bar` as value, doing replace with this global replacer `strings.NewReplacer("/", "_", ".", "_", "-", "_", `\n`,, "\n", `\t`, "\t")` will make the log message as `12_2 foo bar` which is not desired.

Reply:
that is not what I was really asking to use a single replace, but two globals, one for key one for line.

Reply:
you mean we can have 2 separate global variables?

Reply:
yes !

Reply:
Done